cli/src/lib/parseConfig.js

```diff
@@ -43,6 +43,7 @@ const validateConfig = (config) => {
             )
         }
     })
+    // Todo: make sure apps with plugins have `pwa_enabled: true` in the config
```

Comment:
What is the consequence of not checking this yet?

Reply:
Ah this is an obsolete comment -- thanks for catching it! Plugins don't need to enable PWA, but they do if they will be used in PWA parent apps